src/appleseed.studio/mainwindow/rendering/materialdrophandler.cpp

```diff
@@ -0,0 +1,159 @@
+
+//
+// This source file is part of appleseed.
+// Visit https://appleseedhq.net/ for additional information and resources.
+//
+// This software is released under the MIT license.
+//
+// Copyright (c) 2010-2013 Francois Beaune, Jupiter Jazz Limited
```

Comment:
Just like in the header file, replace these two lines with

    // Copyright (c) 2019 Thibault Vergne, The appleseedhq Organization